File: csharp/SwimmingCompetitionCS/services/SwimmingCompetitionException.cs
using System;
using System.Runtime.Serialization;

namespace services
{
    public class SwimmingCompetitionException : Exception
    {
        public SwimmingCompetitionException():base() {}

        public SwimmingCompetitionException(string message) : base(message)
        {
        }

        public SwimmingCompetitionException(string message, Exception innerException) : base(message, innerException)
        {
        }
    }
}

File: java/SwimmingCompetition/SwimmingCompetitionServices/src/main/java/swimmingcompetition/services/SwimmingCompetitionException.java
package swimmingcompetition.services;


public class SwimmingCompetitionException extends Exception{
    public SwimmingCompetitionException() {
    }

    public SwimmingCompetitionException(String message) {
        super(message);
    }

    public SwimmingCompetitionException(String message, Throwable cause) {
        super(message, cause);
    }
}
